pip/requirements.txt

```diff
@@ -214,6 +214,7 @@ neurolab==0.3.5
 node-semver==0.9.0
 notebook==6.5.3
 notebook-shim==0.2.2
+nvidia-ml-py==12.535.133
```

Comment:
@fwyzard , only adding a ackage here will not build/include it for cmssw. Please add it in to https://github.com/fwyzard/cmsdist/blob/IB/CMSSW_14_1_X/master_nvml/python_tools.spec too e.g. `Requires: py3-nvidia-ml-py` so that `cmsBuild` can built it for cmssw-tool-conf

Reply:
Thanks, I'll add that.